<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startMotorsButton">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>110</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start motors</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="filePathTextField">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browseFilesButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>20</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadFileButton">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>20</y>
      <width>99</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>.csv file</string>
    </property>
   </widget>
   <widget class="QLabel" name="cameraImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>500</width>
      <height>281</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-color: #000;
border-style: outset;
border-width: 5px;
border-color: #262626;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Camera image</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileAttributesLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>411</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="fiducialLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>501</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="locateFiducialButton">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>80</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Locate fiducial</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>140</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1129</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
